WoWsShipBuilder.Common/Infrastructure/ApplicationData/DataCacheHelper.cs

```diff
@@ -94,4 +95,59 @@ public static async Task AddToCache(string fileName, string category, string con
 
         Semaphore.Release();
     }
+
+    private static void AddShipToFittingToolSelectorDataStructure(Ship ship)
```

Comment:
check if this method can be changed so it only has to create each dictionary once, for example after downloading the ship data for all ships